app/api/routes/bookmarks.py

```
@@ -0,0 +1,120 @@
+
+class BookmarkOut(BaseModel):
+    id: str
+    user_id: int
+    paper_id: str
+    bookmarked_at: datetime
+    notes: Optional[str] = None
+
+@router.post("", response_model=BookmarkOut, status_code=status.HTTP_201_CREATED)
+def create_bookmark(
+    payload: BookmarkCreate,
+    current_user: User = Depends(get_current_user),
```

Ruff (0.14.5)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)